### ISSUE TITLE
Save state support

### ISSUE THREAD

**Issue by mogtek** [spiritualized1997/openFPGA-Genesis#4]
Are save states currently supported? If not, are there any plans to implement them?

I didn't see them specifically mentioned as unsupported in the README, but they aren't working for me on the Pocket (same error as on cores that specifically say there's no save state support)

**Comment by HappySpaceInvader** [comment]
Yeah, there's definitely no save state support, which is a shame. 

https://github.com/opengateware/openFPGA-Genesis is exactly the same... so I think this is possible a fundamental limitation of this core.
